src/libs/__tests__/Dad.spec.js

```diff
@@ -41,12 +41,36 @@ describe('Dad functionality', () => {
       .toThrowError('Invaild date string format');
   });
 
-  it('if inpunt is second number', () => {
-    const UTC0105 = new Date('2018-01-05 00:00:00+00:00') / 1000;
+  it('if input is second number', () => {
+    const UTC0101 = new Date('2018-01-01 00:00:00+08:00') / 1000;
```

Comment:
It's `+8` the variable's naming  should be `CST0101` would be more clearly.

Reply:
[fixed](https://github.com/17media/dad/pull/14/files#diff-0a42e525241c46c84c5b08c85f34aaf4R45)